app/assets/stylesheets/finder_frontend.scss

```diff
@@ -354,3 +354,9 @@ mark {
   padding-left: 13px;
   padding-right: 13px;
 }
+
+.facets__content {
+  .gem-c-select__select--full-width {
```

Comment:
We shouldn't be styling components in applications - if this is a change that needs to be made to a component, it should be made in the gem, otherwise we're breaking [component isolation](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_principles.md#a-component-is-isolated-when).

Reply:
Thanks, I removed it for now.